src/support.js

```diff
@@ -600,26 +600,38 @@ function makeBigInt(low, high, unsigned) {
   return unsigned ? ((+((low>>>0)))+((+((high>>>0)))*4294967296.0)) : ((+((low>>>0)))+((+((high|0)))*4294967296.0));
 }
 
+/** @param {Array=} args */
+function dynCallDirect(ptr, args) {
+  return wasmTable.get(ptr).apply(null, args);
+}
```

Comment:
this appears in two places for the two runtimes - how about putting it in a single central place? we have a few shared files between them.